splinterd/api/static/openapi.yml

```diff
@@ -469,3 +626,36 @@ components:
         prev: /nodes?offset=0&limit=10
         next: /nodes?offset=20&limit=10
         last: /nodes?offset=40&limit=10
+
+    Notification:
+      type: object
+      properties:
+        hash:
+          type: string
+          example: 1c9b512cef6d6a9064d0cf0eb4b9a957af98bfcec3eb4be2bfb047cf73051cfcca8fa1c142ee12c419d36c55c391b7df38751dae64a1238dfe1e319e9ff22113
```

Comment:
line wrapping (I think you can use folded style: https://symfony.com/doc/current/components/yaml/yaml_format.html#strings)

Reply:
that adds a space for every line break. The other key strings in this file simply exceed the wrap limit